Pear/Controller/TabPageViewController.swift

```diff
@@ -26,7 +26,12 @@ class TabPageViewController: UIPageViewController {
     init(user: UserV2, tabDelegate: TabDelegate) {
         self.user = user
         self.tabDelegate = tabDelegate
-        matchViewController = NoMatchViewController(user: user)
+
+        if let currentMatch = user.currentMatch {
```

Comment:
Is the variable `currentMatch` ever used? If not, let's change this to `if let _ = user.currentMatch` or `if user.currentMatch != nil`